diff --git a/node_modules/@nimiq/core/types/wasm/bundler.d.ts b/node_modules/@nimiq/core/types/wasm/bundler.d.ts
index 7a89e5d..4f83d2e 100644
--- a/node_modules/@nimiq/core/types/wasm/bundler.d.ts
+++ b/node_modules/@nimiq/core/types/wasm/bundler.d.ts
@@ -1,5 +1,12 @@
 /* tslint:disable */
 /* eslint-disable */
+
+declare global {
+    interface SymbolConstructor {
+        readonly dispose: unique symbol;
+    }
+}
+
 /**
  * Available peer service flags
  */
diff --git a/node_modules/@nimiq/core/types/wasm/web.d.ts b/node_modules/@nimiq/core/types/wasm/web.d.ts
index 657e7a2..6407001 100644
--- a/node_modules/@nimiq/core/types/wasm/web.d.ts
+++ b/node_modules/@nimiq/core/types/wasm/web.d.ts
@@ -1,5 +1,12 @@
 /* tslint:disable */
 /* eslint-disable */
+
+declare global {
+    interface SymbolConstructor {
+        readonly dispose: unique symbol;
+    }
+}
+
 /**
  * Available peer service flags
  */
